python/aida_interchange/aifutils.py

```diff
@@ -408,27 +700,41 @@ def mark_informative_justification(g, resource, informative_justification):
     """
     Mark resource as having an informativeJustification value
 
-    :param g: The underlying RDF model for the operation
+    :param rdflib.graph.Graph g: The underlying RDF model
     :param resource: the resource to mark with the specified imporatance
     :param informative_justification: the justification which will be considered informative
     """
     g.add((resource, AIDA_ANNOTATION.informativeJustification , informative_justification))
 
 
 def mark_depends_on_hypothesis(g, depender, hypothesis):
+    """
+    Mark an argument as depending on a hypothesis.
+
+    :param rdflib.graph.Graph g: The underlying RDF model
+    :param rdflib.term.URIRef depender: the argument that depends on the specified hypothesis
+    :param rdflib.term.URIRef hyptothesis: The hypothesis upon which to depend
+    """
     g.add((depender, AIDA_ANNOTATION.dependsOnHypothesis, hypothesis))
 
 
 def mark_as_mutually_exclusive(g, alternatives, system, none_of_the_above_prob):
     """
     Mark the given resources as mutually exclusive.
 
-    :param alternatives: a map from the collection of edges which form a sub-graph for
-    an alternative to the confidence associated with an alternative.
-    :param system: The system object for the system which contains the mutual exclusion
-    :param none_of_the_above_prob: if not None, the given confidence will be applied for
-    the "none of the above" option.
-    :return: The mutual exclusion assertion.
+    This is a special case of [markAsMutuallyExclusive] where the alternatives are 
```

Comment:
Should this be mark_as_mutually_exclusive?

Reply:
yep

Reply:
fixed